### ISSUE TITLE
403 HTTP error on co win API

### ISSUE THREAD

**Issue by vic7z** [vic7z/CovidVaccineNotifier#3]

**Comment by c99SRS** [comment]
HI @vic7z I can work on this. Can you please assign this one to me under hacktoberfest tag?

**Comment by vic7z** [comment]
Yeah sure but this project is long dead and i don't even know the API still works but i will assign you the issue

**Comment by c99SRS** [comment]
HI @vic7z working on this
